### PR TITLE
GET-420 Add new page for registration resend email

### PR DESCRIPTION
**Pull request by cwrw**
jira-ticket: https://dfedigital.atlassian.net/browse/GET-420

Add new page when user clicks resend email on registration. This will continue on to where they clicked register, and show their email.

Also we needed to change the breadcrumbs for the previous confirmation page on registration as well as the title of the page

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,13 +8,22 @@ def create
...
+    render(:registration_results_saved)
+  end
+
+  def registration_send_email_again
+    @user = User.find_or_initialize_by(email: user_params[:email])
+    register_user
+    render(:registration_email_sent_again)
+  rescue NotifyService::NotifyAPIError
+    # TODO: show user an error page, for now render email sent again page
+    render(:registration_email_sent_again)
```
Comment (smoothcontract) [inline app/controllers/users_controller.rb]: Couple of things here perhaps. Firstly, why is the email validation skipped here? If we don't skip that then the body of this method looks almost identical to the one above so potential to DRY this up and just render the appropriate view I think.

Comment (smoothcontract) [reply app/controllers/users_controller.rb]: Discussed in person, the two methods are slightly different, and email validation is not required so not as easy to DRY as it first seems.

Comment (cwrw) [reply app/controllers/users_controller.rb]: talked in person and will clean this up in next ticket

**Comment by cwrw** [comment]
closing in favour of https://github.com/DFE-Digital/get-help-to-retrain/pull/191 which has both changes

**Pull request closed without merging**